lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/nested-journeys/dcmaw-driving-licence.yaml

```diff
@@ -0,0 +1,112 @@
+name: DCMAW and Driving Licence
+description: >-
+  The combined journey for DCMAW and Driving Licence CRIs.
+entryEvents:
+  next:
+    targetState: CRI_DCMAW
+  mitigation-dcmaw:
+    targetState: MITIGATION_01_CRI_DCMAW
+nestedJourneyStates:
+  CRI_DCMAW:
+    response:
+      type: cri
+      criId: dcmaw
+    parent: CRI_STATE
+    events:
+      next:
+        targetState: POST_DCMAW_SUCCESS_PAGE
+      dl-auth-source-check:
+        targetState: CRI_DRIVING_LICENCE_AUTH_SOURCE_CHECK
+      not-found:
+        exitEventToEmit: next-multiple-doc
+      access-denied:
+        exitEventToEmit: next-multiple-doc
+      temporarily-unavailable:
+        exitEventToEmit: next-multiple-doc
+      fail-with-no-ci:
+        exitEventToEmit: next-multiple-doc
+
+  CRI_DRIVING_LICENCE_AUTH_SOURCE_CHECK:
+    response:
+      type: cri
+      criId: drivingLicence
+      context: check_details
+    parent: CRI_STATE
+    events:
+      next:
+        targetState: POST_DCMAW_SUCCESS_PAGE
+      access-denied:
+        targetState: RESET_SESSION_BEFORE_PROVE_ANOTHER_WAY_AFTER_DL
+      alternate-doc-invalid-dl:
+        targetState: RESET_SESSION_BEFORE_DL_ALT_DOC_MITIGATION
+
+  RESET_SESSION_BEFORE_PROVE_ANOTHER_WAY_AFTER_DL:
+    response:
+      type: process
+      lambda: reset-session-identity
+      lambdaInput:
+        resetType: ALL
+    events:
+      next:
+        exitEventToEmit: next-prove-another-way-after-dl
+
+  RESET_SESSION_BEFORE_DL_ALT_DOC_MITIGATION:
+    response:
+      type: process
+      lambda: reset-session-identity
+      lambdaInput:
+        resetType: ALL
+    events:
+      next:
+        exitEventToEmit: next-alternate-doc-invalid-dl-another-way
+
+  POST_DCMAW_SUCCESS_PAGE:
+    response:
+      type: page
+      pageId: page-dcmaw-success
+    events:
+      next:
+        exitEventToEmit: next-address
+
+  MITIGATION_01_CRI_DCMAW:
```

Comment:
This duplication seems a bit suspicious - and indicates that we probably haven't got the abstraction quite right!

What's forcing us to create two copies, and can we avoid it?